packages/scalars/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@powerhousedao/scalars",
+  "version": "0.0.0",
+  "license": "AGPL-3.0-only",
+  "private": false,
+  "files": [
+    "dist"
+  ],
+  "types": "dist/types/src/index.d.ts",
+  "main": "dist/cjs/src/index.js",
+  "module": "dist/es/src/index.js",
+  "type": "module",
+  "scripts": {
+    "check-types": "tsc",
+    "lint": "eslint . && npm run check-types",
```

Comment:
Let's add `"postlint": "npm run check-types"` instead of having it here. This way it's still possible to pass arguments to eslint